homeassistant/components/zwave_js/binary_sensor.py

```diff
@@ -122,13 +126,13 @@ class NewNotificationZWaveJSEntityDescription(BinarySensorEntityDescription):
     NotificationZWaveJSEntityDescription(
         # NotificationType 2: Carbon Monoxide - State Id's 1 and 2
         key=NOTIFICATION_CARBON_MONOOXIDE,
-        states=("1", "2"),
+        states={1, 2},
```

Comment:
I didn't want to change all of these to use constants in this PR. We can do that when we migrate each description to the description in the discovery schema, as seen at the bottom of this module.